### ISSUE TITLE
How to completely disable Albumentations-based augmentations in YOLOv11 (e.g., Blur, MedianBlur etc..)?

### ISSUE THREAD

**Issue by hillsonghimire** [ultralytics/ultralytics#19807]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I tried setting augment=False, but it seems this specific augmentation is still being applied. I couldn’t find clear instructions on how to fully disable it.

`DDP: debug command /user77/miniforge3/envs/user/bin/python -m torch.distributed.run --nproc_per_node 2 --master_port 40121 /user77/.config/Ultralytics/DDP/_temp_073wsljo23452078238096.py
Ultralytics 8.3.86 🚀 Python-3.13.2 torch-2.6.0+cu124 CUDA:0 (NVIDIA A100 80GB PCIe, 81229MiB)
                                                      CUDA:1 (NVIDIA A100 80GB PCIe, 81229MiB)
Overriding model.yaml nc=80 with nc=2
Freezing layer 'model.23.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks...
AMP: checks passed ✅
train: Scanning /path/2.F
train: Caching images (25.5GB Disk): 100%|██████████| 997/997 [00:00<00:00, 704
albumentations: Blur(p=0.01, blur_limit=(3, 7)), MedianBlur(p=0.01, blur_limit=(3, 7)), ToGray(p=0.01, num_output_channels=3, method='weighted_average'), CLAHE(p=0.01, clip_limit=(1.0, 4.0), tile_grid_size=(8, 8))`

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hillsonghimire, thank you for reaching out and for your interest in Ultralytics 🚀! 

We recommend checking out the [Ultralytics Docs](https://docs.ultralytics.com/) for detailed information on training and augmentation settings. You can also explore the [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples to ensure your implementation aligns with our guidelines.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example (MRE)](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate the issue further. This should include a concise script or code snippet that replicates the behavior you’re observing.

For custom training ❓ questions like this one, kindly include as much context as possible, such as your configuration details, relevant code snippets, or any logs that illustrate the behavior. This will help us better understand your query.

To fully explore YOLO11 capabilities and augmentations, you might find [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) helpful as well.

## Upgrade

Please ensure you are using the latest version of the `ultralytics` package to rule out issues that may have already been resolved. You can upgrade by running:

```bash
pip install -U ultralytics
```

## Environments

YOLO11 is verified to run in the following environments, all pre-configured with necessary dependencies like [CUDA](https://developer.nvidia.com/cuda-zone), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM ([GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMI ([AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Image** ([Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Community Support

Join our vibrant Ultralytics community:
- Real-time chat on [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions on [Discourse](https://community.ultralytics.com/)
- Knowledge sharing on our [Subreddit](https://reddit.com/r/Ultralytics)

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests validate YOLO11 operations on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response, but rest assured, an Ultralytics engineer will review your query and respond soon. Thank you for your patience and for being part of the Ultralytics community! 😊

**Comment by Y-T-G** [comment]
You can uninstall albumentations

**Comment by jugal-sheth** [comment]
it did not work i want to get rid of all color changing augmentations i did uninstall sand set individual hsv and every possible thing in args to 0 but cannot set none to auto_augment

I am working on yolov8n-cls but similar issue

**Comment by Y-T-G** [comment]
You can set auto_augment to False

**Comment by glenn-jocher** [comment]
@vittorio-prodomo You can completely disable Albumentations augmentations programmatically by setting the environment variable before importing Ultralytics:

```python
import os
os.environ["ALBUMENTATIONS_DISABLE"] = "1"
from ultralytics import YOLO
```

Alternatively, if you need more granular control over these specific augmentations, you could modify the probability values in the [Albumentations class implementation](https://docs.ultralytics.com/reference/data/augment/#ultralytics.data.augment.Albumentations) by subclassing it, though uninstalling remains the simplest approach for complete removal.
